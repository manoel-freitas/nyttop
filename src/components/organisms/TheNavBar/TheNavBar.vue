<template>
  <NYNavbarRoutes list-class="levelbar" list-item-class="levelbar-item" />
</template>

<script>
import NYNavbarRoutes from 'molecules/NYNavbarRoutes'
export default {
  components: { NYNavbarRoutes }
}
</script>

<style lang="scss">
@import '~bulma/sass/utilities/mixins.sass';
.levelbar {
  display: flex;
  align-items: center;
  padding: 5px 0 4px;
  justify-content: space-evenly;
  border: {
    top: 1px solid #ccc;
  }
  @include mobile {
    flex-direction: column;
    justify-content: space-around;
  }
  &-item {
    button {
      font-size: 0.75rem;
      line-height: 1rem;
      font-weight: 500;
      display: inline-block;
      padding: 0;
      :hover {
        color: black;
      }
    }
  }
}
</style>
