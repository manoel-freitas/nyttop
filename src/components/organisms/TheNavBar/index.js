import TheNavBar from './TheNavBar'

export default TheNavBar
