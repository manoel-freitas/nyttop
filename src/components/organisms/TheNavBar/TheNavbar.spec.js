/// <reference types="jest" />

import { shallowMount } from '@vue/test-utils'
import TheNavBar from './TheNavBar'

const theNavBar = shallowMount(TheNavBar, {
  stubs: ['ny-navbar-item']
})

test('TheNavBar has to be mounted', () => {
  expect(theNavBar.isVueInstance()).toBeTruthy()
})

test('snapshot test', () => {
  expect(theNavBar.element).toMatchSnapshot()
})
