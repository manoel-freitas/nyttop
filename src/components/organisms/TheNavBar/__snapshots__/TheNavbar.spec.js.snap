// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<nynavbarroutes-stub
  listclass="levelbar"
  listitemclass="levelbar-item"
/>
`;
